resources/test/fixtures/flake8_import_conventions/from_imports.py

```diff
@@ -0,0 +1,11 @@
+# Test absolute imports
+import xml.dom.minidom as wrong
+from xml.dom import minidom as wrong
+from xml.dom.minidom import parseString as wrong  # Ensure ICN001 throws on function import.
+
+import xml.dom.minidom as md
+from xml.dom import minidom as md
+from xml.dom.minidom import parseString as pstr
```

Comment:
Maybe add a test for an `import from` with multiple members, only one of which needs to be flagged?

Reply:
Adding test cases was a good call.

Currently, it considers this a violation:
```python
import xml.dom.minidom
```

But it doesn't catch these:
```python
from xml.dom import minidom
from xml.dom.minidom import parseString
```

I'll try to track down what's up. I think these examples should all be violations.